backend/api/settings.py

```diff
@@ -45,8 +45,10 @@
     'django_extensions',
     'corsheaders',
     'tinymce',
+    'storages',
```

Comment:
do you mean `django-storages`?
https://pypi.org/project/django-storages/

Reply:
As unintuitive as it sounds, the package is called django-storages but the installed app is called storages. https://django-storages.readthedocs.io/en/1.1.6/